tests/storage/filesd_metadata_test.py

```diff
@@ -88,20 +88,21 @@
 _SHA_CKSUM=769baef6c65aeef08cf6d177c2c44046b2aac877
 """,
 }
+BACKENDS = userstorage.load_config("storage.py").BACKENDS
```

Comment:
This will work only if the current directory is "tests/storage" is this really true?

Reply:
Yes, tox path defaults to `tests/storage` for the `storage` environment. If we overwrite defaults and select a given test inside this folder, in this case `storage/filesd_metadata_test.py`, it will also work.